ch_pipeline/analysis/flagging.py

```diff
@@ -2517,7 +2542,7 @@ def compute_cumulative_rainfall(
 
     # Load rainfall measurements within relevant time range
     rain_timestamps, rain_meas = load_rainfall(
-        times[0] - accumulation_time_s * 3600 - _TIME_BUFFER,
+        times[0] - accumulation_time_s - _TIME_BUFFER,
```

Comment:
This was already converted to seconds, so we were actually querying a much larger time range than intended